ubuntu24.04-cuda12.8.1-docker28.3.3-python3.12-uv0.8.15.dockerfile

```diff
@@ -151,18 +160,49 @@ COPY config/docker-daemon.json /etc/docker/daemon.json
 # Configure NVIDIA Container Toolkit
 RUN nvidia-ctk runtime configure --runtime=docker
 
+# Copy systemd service files
+COPY config/docker.service /etc/systemd/system/docker.service
+COPY config/docker.socket /etc/systemd/system/docker.socket
+
+# Create docker group
+RUN groupadd -f docker
+
 COPY --from=dind /usr/local/bin/modprobe /usr/local/bin/modprobe
 # Tini (useful if this container is run without a init)
 COPY --from=dind /usr/local/bin/docker-init /usr/local/bin/docker-init
```

Comment:
we can remove this now because we are using systemd and I suppose systemd will collect zombies